### DISCUSSION TITLE
Upload issue results in no error?

### DISCUSSION THREAD

**Discussion by HeartlandTechie** [livewire/livewire#4176] [1. Help]
This is an odd one, but I'll take a crack at it.  I have a Livewire page that I can upload multiple PDF's to . . . then once submitted the page should combine all files into one to allow someone to download.  (Long story . . . ) The issue I can into was on my test server it would only allow about 20 files on a multiple upload and on the production server it was hitting the file limit and the post limit . . . I figured both issues out but the odd one is the failure . . . it never just bombed and gave an error on the Livewire side . . .I guess it wasn't exactly an error per se, but between php.ini settings and Nginx it resulted in a silent failure.  That might be something to tuck away in one's mind during development and error checking on a system . . . in case anyone else runs into this down the road.

**Comment by godismyjudge95** [comment]
Outside of Livewire, I have found upload errors due to memory issues are usually only logged in the laravel logs and/or php error logs.  I would imagine Livewire would not throw any frontward facing error (even in the console) since it (along with Laravel) handles errors more gracefully in production.  If you were to turn on debug mode, it would probably give you the error you were seeking.

**Comment by joshhanley** [comment]
@HeartlandTechie have a look at this discussion #3026, is that the same as what you are seeing?

If so, then yes agreed it should be handled better. Any PR's to improve this would be welcome! 🙂
